src/System.Private.CoreLib/src/System/Globalization/CompareInfo.Unix.cs

```diff
@@ -348,9 +328,37 @@ internal unsafe int GetHashCodeOfStringCore(string source, CompareOptions option
             }
         }
 
-        [DllImport(JitHelpers.QCall)]
-        [SuppressUnmanagedCodeSecurity]
-        private static unsafe extern int InternalHashSortKey(byte* sortKey, int sortKeyLength, [MarshalAs(UnmanagedType.Bool)] bool forceRandomizedHashing, long additionalEntropy);
+        private static unsafe int InternalHashSortKey(byte* sortKey, int sortKeyLength, bool forceRandomizedHashing, long additionalEntropy)
+        {
+            if (forceRandomizedHashing || additionalEntropy != 0)
+            {
+                // TODO: Random hashing is yet to be done
+                // Active Issue: https://github.com/dotnet/corert/issues/2588
+                throw new NotImplementedException();
+            }
+            else
+            {
+
+                // lifted from https://github.com/dotnet/coreclr/blob/master/src/vm/comutilnative.cpp#L3009-L3032
```

Comment:
Nit: A lot of code in CoreRT CoreLib was lifted from somewhere in CoreCLR. We do not include the links to where it was lifted from. (Also, the link you have added will become bogus pretty quickly as the original file will change.)